### DISCUSSION TITLE
Wavy Bonds on unlabled chiral centers

### DISCUSSION THREAD

**Discussion by sudoPete** [rdkit/rdkit#4591] [Q&A] [answered]
Is it possible to draw a wavy bond on unlabeled chiral centers?  I see the function to draw a wavy line, I am just unsure of how to get the atom coords to input into the function to change the line from a straight line to a wavy one.

**Comment by greglandrum** [comment, marked as answer]
At the moment I believe that this isn’t trivial to do (I’m on my phone and can’t easily confirm) but that is definitely something we should add.
